build-logic/src/main/kotlin/polaris-java.gradle.kts

```diff
@@ -204,6 +204,9 @@ tasks.withType<Javadoc>().configureEach {
   val opt = options as CoreJavadocOptions
   // don't spam log w/ "warning: no @param/@return"
   opt.addStringOption("Xdoclint:-reference", "-quiet")
+  if (plugins.hasPlugin("org.kordamp.gradle.jandex")) {
+    dependsOn("jandex")
```

Comment:
Why does javadoc have to depend on jandex? Should it not be the task that packages the jar?

Reply:
Jandex writes to resources, which is "consumed" by javadoc

Reply:
TIL :+1: